infrastructure/dogfood/terraform/aws-tf-module/main.tf

```diff
@@ -460,13 +475,24 @@ module "notify_slack" {
   source  = "terraform-aws-modules/notify-slack/aws"
   version = "5.5.0"
 
-  sns_topic_name = "fleet-dogfood"
+  sns_topic_name = "fleet-dogfood-p1-alerts"
 
   slack_webhook_url = var.slack_webhook
   slack_channel     = "#help-p1"
   slack_username    = "monitoring"
 }
 
+module "notify_slack_p2" {
+  source  = "terraform-aws-modules/notify-slack/aws"
+  version = "5.5.0"
+
+  sns_topic_name = "fleet-dogfood-p2-alerts"
+
+  slack_webhook_url = var.slack_webhook
```

Comment:
We'll need a separate env var for this.  There is already a webhook secret in the repo:

<img width="802" alt="image" src="https://github.com/user-attachments/assets/c3d7d28c-58ab-46ae-9f54-a4f775b957c3">

Once the env var is created, adding another line like https://github.com/fleetdm/fleet/blob/main/.github/workflows/dogfood-deploy.yml#L29 will cover adding the other webhook secret.

Reply:
Great that the P2 webhook exists already!  I updated the TF vars and env vars as indicated, thanks for the callout.